src/components/formElements/MultiselectCustom.vue

```diff
@@ -33,13 +33,21 @@ export default {
       type: Boolean,
       default: true,
     },
+    clearSearchTrigger: Boolean,
   },
   emits: [
     'select',
     'deselect',
     'clear',
     'search-change',
   ],
+  watch: {
+    clearSearchTrigger(value) {
+      if (value) {
+        this.$refs.multiselect.clearSearch();
+      }
+    },
+  },
```

Comment:
does it work correct?
trigger usually means that if it changes it works every time
here it looks like no a trigger, but more like just clearSearch